linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml

```diff
@@ -0,0 +1,21 @@
+<def-group>
+  <definition class="compliance" id="ensure_rtc_utc_configuration" version="1">
+   {{{ oval_metadata("Ensure RTC is using UTC as its time base") }}}
+    <criteria operator="AND">
+      <criterion comment="Check softlink exists for /etc/localtime and look for UTC pattern" test_ref="test_ensure_rtc_utc_configuration" />
+    </criteria>
+  </definition>
+
+  <unix:symlink_test check="all" check_existence="all_exist" id="test_ensure_rtc_utc_configuration" comment="Ensure softlink exist for localtime with UTC pattern" version="1">
+    <unix:object object_ref="object_ensure_rtc_utc_configuration" />
+    <unix:state state_ref="object_ensure_symlink_utc_configuration" />
+  </unix:symlink_test>
```

Comment:
The symlink_test is a new test in OVAL 5.11 test, could you wrap the whole definition in a Jinja conditional?
Similar to https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_sufficiently_large_partition/oval/shared.xml#L1

Reply:
Done, hope it looks right.

Reply:
Apologies for my delay, I believe the version should be just `5.11`.

Reply:
Does this look right? And sorry for my confusion.

Reply:
@brett060102 Yes, thanks.